spec/request/api/v1/user/confirmation_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe 'Confirmation /api/v1/users/confirmation', type: :request do
+  let(:user) { create(:user, confirmed_at: nil) }
+
+  describe 'when signup params are correct' do
```

Comment:
You can also add a test to validate if mail is confirmed. This could be useful:

`expect(user.reload.confirmed?).to be true`